src/AppBundle/Controller/DeleteCounterController.php

```diff
@@ -0,0 +1,198 @@
+
+        // Show the form.
+        return $this->render(
+            'deletionCounter/index.html.twig',
+            [
+            'xtPageTitle' => 'tool-dc',
+            'xtSubtitle' => 'tool-dc-desc',
+            'xtPage' => 'dc',
+            'project' => $theProject,
+            ]
+        );
+    }
+
+    /**
+     * Display the results
+     *
+     * @param string $project  The project domain name.
+     * @param string $username The username.
+     *
+     * @return Response
+     *
+     * @Route("/dc/{project}/{username}", name="DeletionCounterResult")
+     */
+    public function resultAction($project, $username)
+    {
+        /**
+         * Project information
+         *
+         * @var Project $project
+         */
+        $project = ProjectRepository::getProject($project, $this->container);
+        $projectRepo = $project->getRepository();
+
+        $user = UserRepository::getUser($username, $this->container);
+
+        $userID = $user->getId($project);
+
+        if (!$project->exists()) {
+            $this->addFlash('notice', ['invalid-project', $project]);
+            return $this->redirectToRoute('SimpleEditCounter');
+        }
+
+        $dbName = $project->getDatabaseName();
+
+        $loggingTable = $projectRepo->getTableName(
+            $dbName,
+            "logging",
+            "userindex"
+        );
+
+        /**
+         * Connection to the database
+         *
+         * @var Connection $conn
+         */
+        $conn = $this->get('doctrine')->getManager('replicas')->getConnection();
+
+        $types = $this->container->getParameter("deletion_counter");
+
+        if (!isset($types[$project->getDatabaseName()])) {
+            $this->addFlash('notice', [ 'no-result', $username ]);
+            return $this->redirectToRoute(
+                'DeletionCounter',
+                [
+                    'project' => $project->getDomain()
+                ]
+            );
+        }
+
+        $data = $types[$project->getDatabaseName()];
+
+        $queryArray = [];
+
+        foreach ($data as $key => $value) {
+            $queryArray[] = "(SELECT '$key' AS `type`, COUNT(log_id) AS `count`
+            FROM $loggingTable
+            WHERE log_type = 'delete'
+            AND log_user = :userID
+            AND log_comment RLIKE \"$value\")";
+        }
+        $resultQuery = $conn->prepare(join("\nUNION\n", $queryArray));
+
+        $resultQuery->bindParam('userID', $userID);
+        $resultQuery->execute();
+
+        if ($resultQuery->errorCode() > 0) {
+            $this->addFlash('notice', [ 'no-result', $username ]);
+            return $this->redirectToRoute(
+                'DeletionCounterProject',
+                [
+                    'project' => $project->getDomain()
+                ]
+            );
+        }
+
+        $total = 0;
+
+        foreach ($resultQuery->fetchAll() as $row) {
+            $results[$row["type"]] = $row["count"];
```

Comment:
I would agree with this one too (putting `$results = []` somewhere around line 165). Otherwise it would seemingly error out here and on line 174 if `$resultQuery->fetchAll()` doesn't return anything.